target/linux/ath79/files/drivers/mfd/rb4xx-cpld.c

```diff
@@ -21,6 +21,7 @@
 #include <linux/mfd/core.h>
 #include <linux/spi/spi.h>
 #include <linux/module.h>
+#include <linux/of.h>
```

Comment:
ummm I think it's better to remove of_match_ptr. ath79 mandates OF.

Reply:
Fixed. Thank you!